Stratis.Bitcoin.Tests/Converters/BtcDecimalJsonConverterTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using Stratis.Bitcoin.RPC.Converters;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Stratis.Bitcoin.Tests.Converters
+{
+    public class BtcDecimalJsonConverterTest
+    {
+        private readonly ITestOutputHelper _console;
+
+        private readonly JsonSerializerSettings _jsonSerializerSettings = new JsonSerializerSettings
+        {
+            Converters = new List<JsonConverter>
+            {
+                new BtcDecimalJsonConverter()
+            }
+        };
+
+        public BtcDecimalJsonConverterTest(ITestOutputHelper console)
+        {
+            //use this if you want to see the console output for xunit tests.
+            //because xunit runs tests in parallel.
```

Comment:
Ah interesting good to know, can we disable parallel execution on tests (or on project)?